lib/bump.rb

```diff
@@ -61,6 +59,7 @@ def self.bump(file, current, next_version, options)
       commit(next_version, file, options) if options[:commit]
       ["Bump version #{current} to #{next_version}", 0]
     end
+    private_class_method :bump
```

Comment:
wrap everything in `class << self` ?
... diff would be big, but better for the future and with githubs ?w=1 the PR would still look clean ...